Sources/Fullscreen/FullscreenGallery/GalleryPreviewCell.swift

```diff
@@ -56,11 +58,24 @@ class GalleryPreviewCell: UICollectionViewCell {
     override func prepareForReuse() {
         super.prepareForReuse()
         imageView.image = nil
+        imageView.layer.borderWidth = 0.0
     }
 
     // MARK: - Public methods
 
     func configure(withImage image: UIImage?) {
         imageView.image = image
     }
+
+    func border(isVisible: Bool) {
```

Comment:
Functions usually describe an action, so I'd probably call this `showBorder(_ show: Bool)` or something similar.